### PR TITLE
add option to hide 'Add new List'-element

### PR DESCRIPTION
**Pull request by BKDevelop**
This aims to solve #16

What's done:
* implemented feature to hide 'Add another List' element
* added checkbox in plugin options to turn this feature on/off
* placeholder description/titles added
* proper description/title + translations
* picture for plugin options showing the functionality
* add default setting (local storage)

What's missing:

*

I'm not really a JS or web guy, so any c&c is greatly appreciated :)

Thanks in advance,
Ben

Edit:
updated to-do list above

### CODE REVIEW

File: _locales/de/messages.json [hunk 1]
```diff
@@ -27,6 +27,14 @@
     "message": "Fügt einen Button hinzu, der dich bequem den Kompakt-Modus aktivieren/deaktivieren lässt. Wenn aktiv, werden Tags, zugewiesene Menschen, Anhänge und mehr ausgeblendet. so kannst Du dich auf das konzentrieren, was wirklich wichtig ist.",
     "description": "Description of the Compact Mode feature."
   },
+  "settingsHideAddNewListTitle": {
+    "message": "Verstecke \"Eine weitere Liste hinzufügen\"",
+    "description": "Name of the 'Hide Add New List' feature."
+  },
+  "settingsHideAddNewListDescription": {
+    "message": "Versteckt den \"Eine weitere Liste hinzufügen\"-Button",
```
Comment (christiankaindl) [inline _locales/de/messages.json]: I think something more descriptive would be appropriate. Maybe something along these lines: 

**en**: "This hides the additional column with the "Add new list" button and saves horizontal space. You can still create new lists either by double clicking on an empty area on the board or by using the keyboard shortcut _XY_"

What I want to achieve by this message is 1) that people understand what column is being meant (the image will help with that) and what is the advantage to disabling the button (less horizontal scrolling), and 2) that people can learn how to create a new list even without the button (double click & shortcut).

(I don't know the shortcut right now, but I am pretty sure there is one. The correct shortcut should make it into the message above)

File: _locales/de/messages.json [hunk 2]
```diff
@@ -27,6 +27,14 @@
     "message": "Fügt einen Button hinzu, der dich bequem den Kompakt-Modus aktivieren/deaktivieren lässt. Wenn aktiv, werden Tags, zugewiesene Menschen, Anhänge und mehr ausgeblendet. so kannst Du dich auf das konzentrieren, was wirklich wichtig ist.",
     "description": "Description of the Compact Mode feature."
   },
+  "settingsHideAddNewListTitle": {
+    "message": "Verstecke \"Eine weitere Liste hinzufügen\"",
+    "description": "Name of the 'Hide Add New List' feature."
+  },
```
Comment (christiankaindl) [inline _locales/de/messages.json]: The englisch strings are missing

**Review by christiankaindl — COMMENTED** [review]
Thanks for your Pull Request! I looked at it a bit and noted some feedback :)

Before we can merge:
- In background.js the default check for the new setting is still missing (should default to `false`): https://github.com/christiankaindl/trello-super-powers/blob/6c568ff6bccfeef01b919ae447fd679cc24015a6/background.js#L40
- English translation strings

After that we can merge and I'll test it out for a few days.

**Comment by BKDevelop** [comment]
Thanks for your comments! I really like your approach to writing the description.

I was away over the weekend but I'll try to replace the placeholders this week. :)

**Comment by christiankaindl** [comment]
I'll have a look at it this week, thanks!

**Comment by BKDevelop** [comment]
I just realised I forgot to mention something: I checked Trello documentation and some Google results for a keybord shortcut to add another list. However I was not able to find one, so I left out the corresponding part in the descriptions!

Have a nice week

**Review by christiankaindl — COMMENTED** [review]
Very nice, thanks! The only thing that needs correcting in my opinion is the screenshot.

I'd suggest something like the following, as it gives more context to the user:
grafik

After that it's ready. I'll test out the changes and release it ASAP :)

**Comment by BKDevelop** [comment]
I hope you like that screenshot better :)

**Review by christiankaindl — APPROVED** [review]

**Comment by orngreen** [comment]
Sorry if this is a stupid question, but I have zero knowledge regarding building Firefox extensions.

Seeing that you merged this feature to master, when do you believe the update will be included from:
https://addons.mozilla.org/en-US/firefox/addon/trello-super-powers/

Because I would really, really like this feature :) :)

**Comment by christiankaindl** [comment]
Yeah, GitHub is just the code repository. The release process is manual. I still need to test the changes on my machine, then I can make it available to all users.

I'll do it later today and post here when it's available for you 👍

**Comment by christiankaindl** [comment]
@BKDevelop @orngreen 

The new version is now available at https://addons.mozilla.org/en-US/firefox/addon/trello-super-powers/ 

In my testing everything worked. If you experience any issues, please reach out. And thanks @BKDevelop for developing this feature!

---

_You can update via Firefox's Add-on page (type about:addons in the url bar, or click on Firefox's menu in the top right and choose "Add-ons")._

**Pull request merged**